spec/order_spec.rb

```diff
@@ -0,0 +1,75 @@
+require 'order'
+require 'stringio'
+
+describe Order do
+  let(:menu) do
+    double :menu,
+    menu: [{dish: 'BBQ',       price: 10},
+           {dish: 'Hamburger', price: 8},
+           {dish: 'Pizza',     price: 9},
+           {dish: 'Pasta',     price: 7},
+           {dish: 'Sushi',     price: 6}]
+  end
+  describe "#greet" do
+    it "greets to the customer" do
+      expect { subject.greet }.
+        to output("Thank you for visiting our takeaway website.\n" +
```

Comment:
Style/LineEndConcatenation: Use `\` instead of `+` or `<<` to concatenate those strings.